<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUIClass</class>
 <widget class="QMainWindow" name="MainUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>804</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>joy</string>
  </property>
  <widget class="QWidget" name="bodyWidget">
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>804</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>330</width>
     <height>804</height>
    </size>
   </property>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-3</y>
      <width>328</width>
      <height>802</height>
     </rect>
    </property>
    <widget class="QWidget" name="roonListWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>330</width>
       <height>689</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="liveRoomListBtn">
      <property name="geometry">
       <rect>
        <x>302</x>
        <y>17</y>
        <width>13</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/mainui/flush_button);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="roomlistWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>310</width>
        <height>643</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="liveRoomListLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>主播</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget1" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>330</width>
        <height>44</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>44</y>
        <width>10</width>
        <height>643</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>44</y>
        <width>10</width>
        <height>643</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <zorder>roomlistWidget</zorder>
     <zorder>widget1</zorder>
     <zorder>liveRoomListBtn</zorder>
     <zorder>liveRoomListLabel</zorder>
     <zorder>widget</zorder>
     <zorder>widget_2</zorder>
    </widget>
    <widget class="QWidget" name="appInfoWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>330</width>
       <height>44</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="closeBtn">
      <property name="geometry">
       <rect>
        <x>303</x>
        <y>13</y>
        <width>15</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/joy/close_button);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="minBtn">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>13</y>
        <width>15</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/joy/min_button);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="appLogoGraphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>173</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="languageBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>13</y>
        <width>25</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>中</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="userInfoWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>49</y>
       <width>330</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:0, stop:0 rgba(244, 248, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <widget class="QLabel" name="userUidLabel">
      <property name="geometry">
       <rect>
        <x>64</x>
        <y>15</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="appVersionLabel">
      <property name="geometry">
       <rect>
        <x>64</x>
        <y>38</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V1.0.0, #3, TB:2.9.10, ChatRoom:1.4.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="liveBtn">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>12</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/mainui/begin_living_button);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="userAvatorGraphicsView">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>44</width>
        <height>44</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/mainui/cover_default);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
</ui>
